keyboards/ut47/keymaps/Nordic/keymap.c

```diff
@@ -0,0 +1,277 @@
+/* Copyright 2018 Carlos Filoteo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#ifdef LED_ENABLE
+  #include "protocol/serial.h"
+
+#ifndef KEYMAP_NORDIC_H
```

Comment:
You shouldn't need all of this.  A simple `#include "keymap_nordic.h"` should handle all of this. 

If not, then it may be worth creating a `keymap_nordic.h` in the keymap folder, and then use that, until the main nordic file can be changed.